evm/src/cpu/kernel/ast.rs

```diff
@@ -29,6 +29,7 @@ pub(crate) enum Item {
 pub(crate) enum PushTarget {
     Literal(Literal),
     Label(String),
+    Var(String),
```

Comment:
nit: Maybe call it `MacroVar` to make it clear that it should only be used in macros, or add a comment.